test/unit/lib/isNotEnglishLocation.js

```diff
@@ -0,0 +1,72 @@
+const isNotEnglishLocation = require('../../../app/lib/isNotEnglishLocation');
+const chai = require('chai');
+
+const expect = chai.expect;
+
+describe('isNotEnglishLocation', () => {
```

Comment:
Has the test for `IM` postcode and one ending in `IM` been purposefully omitted?